### ISSUE TITLE
Page not found

### ISSUE THREAD

**Issue by acastillopavia** [AsgardCms/Platform#201]
### Summary of issue
I installled  it from the scratch. The server uses userdir apache module, exemple http://localhost/~user/projectname

1. move all files from public to folder project
2. add line to  .htaccess
RewriteBase /~user/projectname/  
3. change url variable file config/app.php  
http://localhost/~user/projectname
4. change file index.php in home folder
  require __DIR__.'../bootstrap/autoload.php';
  to
  require __DIR__.'/bootstrap/autoload.php';

Backend and frontend works fine at this point.
Once I logged in the backend  and update some content the home page in the website crashes 

**NotFoundHttpException in Application.php line 879:**

in Application.php line 879
at Application->abort('404') in PublicController.php line 72
at PublicController->throw404IfNotFound(null) in PublicController.php line 47
### System Details

Linux Fedora 23
PHP 5.6.22
Laravel 5

Ever
- Operating System
- PHP Version
- Laravel Version
- `Asgardcms/Platform` version

**Comment by nWidart** [comment]
Hello,

This is most likely because the CMS wasn't built to handle this use case of folder structure.
You'd need to create a vhost pointing to the public directory in order to make it work.

**Comment by joshbrw** [comment]
To iterate what @nWidart said.. this isn't the directory structure that Asgard (or Laravel) was built for, so errors are likely

**Comment by acastillopavia** [comment]
Thank you for your answers. I made it in the good way this time (no changes in the folder structure) but it's giving me the same error. I've installed the website and it is working properly. I logged in the administrator area and it is ok. I modify a page clicking on the update button and when I want to see the modification in the website it's giving me the same error than before...
-----------------------------------------------------------------------
Sorry, the page you are looking for could not be found.

NotFoundHttpException in compiled.php line 1864:
in compiled.php line 1864
at Application->abort('404') in PublicController.php line 72
-------------------------------------------------------------------------

Thank you very much for your help
